### PR TITLE
[13.0][MIG] sale_timesheet_task_exclude

### PR DESCRIPTION
**Pull request by flachica**

### CODE REVIEW

File: sale_timesheet_task_exclude/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Sales Timesheet: exclude Task from Sale Order",
+    "version": "13.0.1.0.0",
+    "category": "Sales",
+    "website": "https://github.com/OCA/timesheet",
+    "author": "Brainbean Apps, " "Odoo Community Association (OCA)",
```
Comment (davejames) [inline sale_timesheet_task_exclude/__manifest__.py]: black has left some extra double quotes

**Review by davejames — CHANGES_REQUESTED** [review]

File: sale_timesheet_task_exclude/__manifest__.py (hunk 1 above)
Comment (flachica) [reply sale_timesheet_task_exclude/__manifest__.py]: Maybe. I forced push

**Comment by davejames** [comment]
code LGTM, there are some administrative commits that should be squashed according to the migration guidelines

**Comment by flachica** [comment]
> code LGTM, there are some administrative commits that should be squashed according to the migration guidelines

I cleaned a bit

**Comment by davejames** [comment]
I think something is wrong with the git history, it looks like it wants to merge a whole bunch of commits which are already in the 13.0 branch, maybe a rebase will fix it?

**Comment by flachica** [comment]
> I think something is wrong with the git history, it looks like it wants to merge a whole bunch of commits which are already in the 13.0 branch, maybe a rebase will fix it?

Excuse me, I just fixed it

**Review by davejames — APPROVED** [review]
Code review, LGTM

**Review by alexey-pelykh — APPROVED** [review]

**Pull request merged**